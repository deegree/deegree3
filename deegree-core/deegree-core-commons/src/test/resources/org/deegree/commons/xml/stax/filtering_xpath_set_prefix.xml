<?xml version='1.0' encoding='UTF-8'?>
<app:a xmlns:app="http://www.deegree.org/app">
  <nix:c xmlns:nix="http://www.deegree.org/nix" />
  <nix:d xmlns:nix="http://www.deegree.org/nix" />
</app:a>
