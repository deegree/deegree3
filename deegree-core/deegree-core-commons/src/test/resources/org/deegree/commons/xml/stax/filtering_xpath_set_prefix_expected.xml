<app:a xmlns:app="http://www.deegree.org/app">
  
  
  <nix:d xmlns:nix="http://www.deegree.org/nix"/>

</app:a>